Content.Server/Explosions/ExplosionHelper.cs

```diff
@@ -76,29 +77,33 @@ public static void SpawnExplosion(GridCoordinates coords, int devastationRange,
             foreach (var tile in tiles)
             {
                 var tileLoc = mapGrid.GridTileToLocal(tile.GridIndices);
-                var tileDef = (ContentTileDefinition)tileDefinitionManager[tile.Tile.TypeId];
-                var distanceFromTile = (int)tileLoc.Distance(mapManager, coords);
-                if (!string.IsNullOrWhiteSpace(tileDef.SubFloor))
+                var tileDef = (ContentTileDefinition) tileDefinitionManager[tile.Tile.TypeId];
+                ContentTileDefinition.BaseTurfs.TryGetValue(tileDef.Name, out var baseTurfs);
+                var distanceFromTile = (int) tileLoc.Distance(mapManager, coords);
+                if (!baseTurfs.Any())
```

Comment:
```suggestion
                if (baseTurfs.Count == 0)
```

Using `.Any()` is significantly slower since it's forced to treat the collection like a plain `IEnumerable`, allocating in the process.